spec/qiita/markdown/processor_spec.rb

```diff
@@ -978,7 +978,17 @@
         end
 
         it "returns code element with its color" do
-          should eq "<p><code>#FF0000<span class=\"\" style=\"background-color: #FF0000;\"></span></code></p>\n"
+          should eq "<p><code>#FF0000<span class=\"inline-code-color\" style=\"background-color: #FF0000;\"></span></code></p>\n"
+        end
+
+        context "with class name of inline code color parameter" do
+          let(:context) do
+            { hostname: "example.com", inline_code_color_class_name: "qiita-inline-code-color" }
```

Comment:
The rspec context mentions about `:inline_code_color_class_name` but the `let` specifies `:hostname` too. It should represents the context:

```rb
let(:context) do
  super().merge(inline_code_color_class_name: 'qiita-inline-code-color')
end
```

Reply:
Exactly. I fixed it in 4e8f3f02c86e391b96c60a24e930c6f8cc1ba4d5 .